src/test/java/com/google/javascript/clutz/undefined_provide.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace ಠ_ಠ.clutz_internal.undefined_provide {
+}
+declare module 'goog:undefined_provide' {
+  import alias = ಠ_ಠ.clutz_internal.undefined_provide;
+  export = alias;
+}
```

Comment:
At first I thought this is not emitting `goog.require` shim, but with our change to have default `goog.require` be empty interface, the default type is correct. Carry on.